charts/zookeeper-operator/templates/operator.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
+      {{- if .Values.additionalSidecarVolumes }}
+      volumes:
+{{- include "chart.additionalSidecarVolumes" . | indent 6 }}
```

Comment:
You can pass `.Values.additionalSidecarVolumes` to the `chart.additionalSidecarVolumes` template or use with to change the 'dot' 
```suggestion
      {{- with .Values.additionalSidecarVolumes }}
      volumes:
{{- include "chart.additionalSidecarVolumes" . | indent 6 }}
```

Reply:
no longer needed

Reply:
no longer needed